zhaquirks/tuya/ts110e.py

```diff
@@ -0,0 +1,142 @@
+"""Tuya Dimmer TS110E."""
+from typing import Optional, Union
+
+from zigpy.profiles import zha
+import zigpy.types as t
+from zigpy.zcl import foundation
+from zigpy.zcl.clusters.general import (
+    Basic,
+    GreenPowerProxy,
+    Groups,
+    LevelControl,
+    OnOff,
+    Ota,
+    Scenes,
+    Time,
+)
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.tuya import TuyaDimmerSwitch, TuyaZBExternalSwitchTypeCluster
+
+TUYA_LEVEL_ATTRIBUTE = 0xF000
+TUYA_CUSTOM_LEVEL_COMMAND = 0x00F0
+
+
+class TuyaLevelPayload(t.Struct):
+    """Tuya Level payload."""
+
+    level: t.uint16_t
+    transtime: t.uint16_t
```

Comment:
It should probably move to `tuya.__init__.py`, but let's keep it here for now.

Reply:
Or first do the implementation in the quirk and getting it merged  and its working OK and then making one more PR for moving it to tuya INIT then tuya INIT is little tricky with many addings and can needing rebasing many times and its PITA (i have one PR waiting for being merged to it).

Reply:
Yes, that's the idea.
Keep here and merge. When required, move to Tuya INIT. 👍🏻